### PR TITLE
chore(helm): add possibility to expose custom ports on Service

### PR DESCRIPTION
**Pull request by Mopsgeschwindigkeit**
Currently it is not possible to expose custom ports on the Service created by helm. Therefore, if you want to call a custom listener, listening on a port different as the default ports, you can't access them using the Service created by helm. This PR will conflict with #11375.

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [ ] Added tests for the changes
- [ ] Added property-based tests for code which performs user input validation
- [ ] Changed lines covered in coverage report
- [ ] Change log has been added to `changes/(ce|ee)/(feat|perf|fix|breaking)-<PR-id>.en.md` files
- [ ] For internal contributor: there is a jira ticket to track this change
- [ ] Created PR to [emqx-docs](https://github.com/emqx/emqx-docs) if documentation update is required, or link to a follow-up jira ticket
- [ ] Schema changes are backward compatible

## Checklist for CI (.github/workflows) changes

- [ ] If changed package build workflow, pass [this action](https://github.com/emqx/emqx/actions/workflows/build_packages.yaml) (manual trigger)
- [ ] Change log has been added to `changes/` dir for user-facing artifacts update

### CODE REVIEW

**Comment by Mopsgeschwindigkeit** [comment]
@Rory-Z Hello :)

Is there a problem with this PR ? If yes, let me know, so i can fix 👍🏾

**Comment by Rory-Z** [comment]
> @Rory-Z Hello :)
> 
> Is there a problem with this PR ? If yes, let me know, so i can fix 👍🏾

@Mopsgeschwindigkeit Thanks for your code, all of looks good, could you please fix conflicts ?

@id What do you think about `customPorts` as field name ?

File: deploy/charts/emqx-enterprise/README.md [hunk 1]
```diff
@@ -68,11 +68,13 @@ The following table lists the configurable parameters of the emqx chart and thei
 | `service.ws`                         | Port for WebSocket/HTTP.                                                                                                                                     | 8083                                                    |
 | `service.wss`                        | Port for WSS/HTTPS.                                                                                                                                          | 8084                                                    |
 | `service.dashboard`                  | Port for dashboard and API.                                                                                                                                  | 18083                                                   |
+| `service.customPorts`                | Custom Ports to exposed in the Service.                                                                                                                      | {}                                                      |
```
Comment (id) [inline deploy/charts/emqx-enterprise/README.md:71]:
```suggestion
| `service.customPorts`                | Custom Ports to be exposed in the Service.                                                                                                                      | {}                                                      |
```

File: deploy/charts/emqx/README.md [hunk 2]
```diff
@@ -69,11 +69,13 @@ The following table lists the configurable parameters of the emqx chart and thei
 | `service.ws`                         | Port for WebSocket/HTTP.                                                                                                                                     | 8083                                                    |
 | `service.wss`                        | Port for WSS/HTTPS.                                                                                                                                          | 8084                                                    |
 | `service.dashboard`                  | Port for dashboard and API.                                                                                                                                  | 18083                                                   |
+| `service.customPorts`                | Custom Ports to exposed in the Service.                                                                                                                      | {}                                                      |
```
Comment (id) [inline deploy/charts/emqx/README.md:72]:
```suggestion
| `service.customPorts`                | Custom Ports to be exposed in the Service.                                                                                                                      | {}                                                      |
```

**Comment by id** [comment]
Looks good to me. Added a small syntax remark.

**Comment by Mopsgeschwindigkeit** [comment]
@Rory-Z Conflicts are resolved.

**Review by Rory-Z — DISMISSED** [review]

**Comment by Rory-Z** [comment]
Hi @Mopsgeschwindigkeit Could you please fix the failed check? looks the commit format is the reason of the failed check

**Comment by Mopsgeschwindigkeit** [comment]
@Rory-Z I squashed the the "Apply Suggestion" commit into my original one. I hope the Merge-Commit does break the action either.

**Comment by Mopsgeschwindigkeit** [comment]
6/8 test failed because the Chart.yam inside `deploy/charts/emqx/` directory was missing. Merged upstream/master again.
For the other 2 failing tests, i have no idea.

**Comment by id** [comment]
@Mopsgeschwindigkeit cherry-picked your commit on top of latest master to keep the commit history simple. You are listed as an author, and I am as a committer for that one. Also moved back Chart.yaml.

**Review by Rory-Z — APPROVED** [review]

**Comment by id** [comment]
@Mopsgeschwindigkeit this will be released in 5.9.0.

**Comment by emqxqa** [comment]
[TestExecution](https://emqx.atlassian.net/browse/QA-6673)

**Pull request merged**